pkg/convert/testdata/programs/object_keys_renamed/main.tf

```diff
@@ -0,0 +1,14 @@
+locals {
```

Comment:
I think this test is valid. We should be able to raise a PR with just this new test, and without any other changes to the other generated files.